packages/web/components/trade-clipboard/index.tsx

```diff
@@ -0,0 +1,481 @@
+import React, { useEffect, useMemo, useRef } from "react";
+import Tippy from "@tippyjs/react";
+import { TokenSelect } from "../token-select";
+import {
+  useBooleanWithWindowEvent,
+  useSlippageConfig,
+  useTradeTokenInConfig,
+} from "../../hooks";
+import classNames from "classnames";
+import { Pool } from "@osmosis-labs/pools";
+import { observer } from "mobx-react-lite";
+import { useStore } from "../../stores";
+import AutosizeInput from "react-input-autosize";
+import { Currency } from "@keplr-wallet/types";
+
+export const TradeClipboard = observer<
+  {
+    containerClassName?: string;
+    containerStyle?: React.CSSProperties;
```

Comment:
Should we use `ClassNames` for passing tailwind classes instead?

Reply:
I made this bc there were certain styles that cannot be passed on using tailwind.